index.html

```diff
@@ -0,0 +1,245 @@
+                     </li>
+                    
+                     <li><a>English</a></li>
+                     
+                     <li><a>My page</a></li>
+                     
+                     <li>Partners</li>
+                </ul>
+            </div>
+
+            <!--Nav Pc ESCRITORIO-->
+             <div class="container">
+
+                 <button id="mobileButton"></button>
+
+                 <a href="index.html">
+                    <img class="mylogo" src="./images/company.png">
+                 </a>
+
+                 <ul>
+                     <li>
+                         <a href="about/about.html">About</a>
+                     </li>
+ 
+                     <li><a href="#containercards">Program</a></li>
+ 
+                     <li>
+                         <a href="#people">Speakers</a>
+                     </li>
+                    
+                     <li>
+                         <a href="#sponsor">Sponsors</a>
+                     </li>
+ 
+                     <li id="about">
+                        <a href="about/about.html"> CC Campaign
+                        </a>
+                     </li>
+                 </ul>
+             </div>
+        </nav>
+
+     </section>
+
+     <!--presentation-->
+     <section class="presentation">
+
+         <section class="allpresentation">
+
+             <!--Presentacion-->
+             <div>
+
+                <span class="smalltitle">"Hello Sharing world!"</span><!--New class-->
+
+                <h1>COMPUTER MUSEUM<br> COMPANY</h1><!--New class-->
+
+                 <div class="info">
+
+                    <p>
+                        A joyful celebration believing in the value of openness and sharing, creating <br>
+                        a positive change with people from over 80 countries is taking place in July, In Argentina.
+                    </p>
+
+                 </div>
+                
+                 <h2>Since 2023.7.23(THU) - 24(FRI)</h2>
+                
+                 <h4 class="subdescription">@National Museum of Argentina, Computer Museum.</h4>
+             </div>
+
+         </section>
+
+     </section>
+
+     <!--Cards-->
+     <section id="containercards">
+
+        <h3> Main Program</h3>
+
+        <hr>
+
+        <section class="cards">
+            
+            <section class="individualcard">
+
+                <div class="card3">
+
+                     <h4 class="lectur">Lecture</h4>
+                     <p class="info">Listen to the speakers from various countries about the messages of sharing and opening</p>
+
+                </div>
+
+            </section>
+
+            <section class="individualcard">
+                <div class="card3">
+                     <h4 class="catalog1">CC Exhibition</h4>
+                     <p class="info2">Get to know the new and interesting exhibitions made by our company. </p>
+                 </div>
+            </section>
```

Comment:
- [ ] Great work here 👏 . However, remember the essence of the project is to have an exact replica as much as possible.
- On that note, kindly adjust the middle section to tally with the layout given on the design.
- individual articles have their elements organised in an horizontal row, with the image at the left, title in the center, and description at the right.

### Your view:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/53536426-a69f-4b15-b476-b89d711b01fa)
### Design given:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/40e86157-4fd0-43eb-af09-f2aed5fc92e4)

Reply:
Thank you very much for the suggestion, the changes have been made satisfactorily 👍😊